safe/common/test/test_version.py

```diff
@@ -58,8 +58,9 @@ def test_get_version(self):
     def test_get_current_hash(self):
         """Test for get_current_hash."""
         git_hash = current_git_hash()
-        if 'win32' in sys.platform:
+        if 'win32' in sys.platform or 'darwin' in sys.platform:
```

Comment:
It was `'win32' in sys.platform`, so the sys.platform has substring `win32`. So, it can be failed if I check the reverse you suggested. I don't know the output of sys.platform in Windows, so just making sure.